data/Debian-family.yaml

```diff
@@ -0,0 +1,3 @@
+---
+letsencrypt::configure_epel: false
```

Comment:
Why don't you create a `common.yaml` to set this to avoid setting it for every OS (except RH)?

Reply:
it is now with `common.yaml`